ge.c

```diff
@@ -271,7 +271,6 @@ main(int argc, char **argv)
 	setlocale(LC_CTYPE, "");
 
 	log_init(1, LOG_DAEMON);
-	log_setverbose(0);
```

Comment:
why this change?

Reply:
I figured if `gemexp` is for experimenting, a verbose log output might be useful as well. This was because of the `log_debug()` when we parsed the proxy protocol.

I can revert this, no problem. I though that this might be something to consider, though.